server/middleware.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from "express";
+
+import config from "./config";
+
+export function isAdmin(
+	request: express.Request,
+	response: express.Response,
+	next: express.NextFunction
+) {
+	response.setHeader("Cache-Control", "private");
+	const auth = request.headers.authorization;
+
+	if (auth && typeof auth === "string" && auth.indexOf(" ") > -1) {
+		const key = new Buffer(auth.split(" ")[1], "base64").toString();
```

Comment:
Is this deprecated? https://nodejs.org/api/buffer.html#buffer_new_buffer_string_encoding

Reply:
good catch